language/stdlib/transaction_scripts/rotate_authentication_key_with_nonce_admin.move

```diff
@@ -60,6 +61,8 @@ spec fun rotate_authentication_key_with_nonce_admin {
         Errors::NOT_PUBLISHED; // TOOD: Undocumented error code. Added due to the possible absence of SlidingNonce in SlidingNonce::try_record_nonce;
 
     /// Access Control
+    /// Only the Libra Root account can process the admin scripts [[H8]][PERMISSION].
+    requires Roles::has_libra_root_role(lr_account); /// This is ensured by LibraAccount::writeset_prologue.
```

Comment:
Would it make more sense to require the entire `spec fun` for `writeset_prologue` here? That would make the comment unnecessary + let the script assume other properties that are true and may be needed in the future.

Reply:
That's a good point. Thanks to @emmazzz , each txn script already has the following line in order to "require" other properties that the prologue function ensures:
```
include LibraAccount::TransactionChecks{sender: account}; // properties checked by the prologue.
```

Reply:
The problem is that admin scripts *do not* run the ordinary transaction prologue, so it is not correct to assume this. Instead, admin scripts should assume that the writeset prologue is run.